_posts/2018-06-13-get-ready-to-rebble.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Getting Ready for Rebble"
```

Comment:
Is it `Getting Ready for Rebble` or `Get Ready to Rebble` per the file name?  I kind of love the latter TBH! :bowtie:

Reply:
I like the latter more but it makes less sense, so I left it in the slug as an Easter egg. :p

Reply:
@Katharine ha, love it! :bowtie: 🔎 🥚